src/SX1276fsk.cpp

```diff
@@ -336,7 +336,7 @@ int SX1276fsk::receive(void* ptr, int len) {
         printf("SX1276fsk: RX restart (RSSI thres is %ddBm)\n", -readReg(REG_RSSITHRES)/2);
 
     // read RSSI and do some smoothed tracking of background noise
-    } else if (!synAddrMatch && uNow-bgRssiAt > 10*1000) {
+    } else if (rssiAt == 0 && !synAddrMatch && uNow-bgRssiAt > 10*1000) {
```

Comment:
I am actually not sure. I wanted to prevent reading bgRSSI when RSSI is detected, but not yet SYNCADDRESS matched. 

However, when current RSSI threshold is too low, this may break the purpose. It only samples bgRSSI when the threshold is not exceeded.